content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -190,32 +203,27 @@ You will be prompted with configuration options:
 Site Created
 
 Admin URL: https://app.netlify.com/sites/YOUR_SITE_NAME
-URL:       https://YOUR_SITE_NAME.netlify.com
+URL:       https://YOUR_SITE_NAME.netlify.app
 ```
 
 > **Note:** You can change the build configuration via the `netlify.toml` file and the site name from the Netlify admin UI.
 
-## 9. Expose the DATABASE_URL environment variable to functions
-
-Since Prisma will connect to the database from your serverless functions, you need to expose the `DATABASE_URL` to the backend functions. This is typically done from the Netlify UI. To do, open the Netlify admin UI for the site. You can do so directly with the Netlify CLI as follows:
-
-```terminal
-netlify open --admin
-```
-
-Once open, go to **Site settings**:
-
-![Netlify admin UI](https://res.cloudinary.com/prismaio/image/upload/v1628765152/docs/p472v73_amyavf.png)
+## 8. Expose the <inlinecode>DATABASE_URL</inlinecode> environment variable to functions
 
-Navigate to **Build & deploy** in the sidebar on the left and then select **Environment**:
+Because Prisma will connect to the database from your serverless functions, you need to expose the `DATABASE_URL` to the backend functions. This is typically done from the Netlify UI.
 
-![Netlify environment variables](https://res.cloudinary.com/prismaio/image/upload/v1628765166/docs/UJc23f8_jfpxmf.png)
+1. Open the Netlify admin UI for the site.
+   You can do so directly with the Netlify CLI as follows:
+   ```terminal
+   netlify open --admin
+   ```
+2. Click **Site settings**:
 
-Create a variable with the key `DATABASE_URL` and set the value with your PostgresSQL URL.
+![Netlify admin UI](https://res.cloudinary.com/prismaio/image/upload/v1628765152/docs/p472v73_amyavf.png) 3. Navigate to **Build & deploy** in the sidebar on the left and select **Environment**:
 
-> **Note:** It's considered best practice to keep secrets out of your codebase. If you open up the `prisma/schema.prisma` file, you should see `env("DATABASE_URL")` in the _datasource_ block. By setting an environment variable you keep secrets out of the codebase.
+![Netlify environment variables](https://res.cloudinary.com/prismaio/image/upload/v1628765166/docs/UJc23f8_jfpxmf.png) 4. Create a variable with the key `DATABASE_URL` and set its value to your database connection string.
```

Comment:
The ordered list breaks here:
<img width="422" alt="image" src="https://user-images.githubusercontent.com/183673/170079088-6dcbcd0f-3c73-44cd-8970-192a0dea3e6c.png">

(Images are invisible in the PErcy screenshot)

Getting that right iwll probaly require removing the empty lines and finding the correct indentation for the images that should be "in" the list so Markdown realizes that this is all one big list.

Sometimes it is easier to fall back to HTML (`<ol><li>...</li>...</ol>`) in these cases.